.github/workflows/release-dotnet.yml

```diff
@@ -0,0 +1,42 @@
+name: Release .NET
+
+on:
+  push:
+    branches: [ main ]
+
+env:
+  PROJECT_PATH: packages/dotnet/Cisco.Opentelemetry.Specifications
+  CSPROJ_PATH: packages/dotnet/Cisco.Opentelemetry.Specifications/Cisco.Opentelemetry.Specifications.csproj
+  PACKAGE_OUTPUT_DIRECTORY: ${{ github.workspace }}/output
+  NUGET_SOURCE_URL: https://api.nuget.org/v3/index.json
+
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v2
+        with:
+          dotnet-version: 6.0.x
+          
+      - name: Setup NodeJs
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
+      - name: Install dotnet-bump
+        run: npm install -g dotnet-bump
+            
+      - name: Bump version
+        working-directory: ${{ packages/dotnet/Cisco.Opentelemetry.Specifications }}
+        run: dotnet-bump patch --dry-run --verbose
+      
+#      - name: Pack & Publish
+#        run: |
+#          dotnet clean
```

Comment:
Do we have an option to run it with dry run?

Reply:
There's no dry run for the publish command